submissions/EnF1NeeT/HTML-CSS_popup/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Alumni+Sans:ital,wght@0,300;1,300&display=swap" rel="stylesheet">
+    <title>Kottanspost</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="nav-list list">
+          <li class="nav-item">
+            <input type="checkbox" id="menu-check" class="input-check">
+            <label for="menu-check" class="check"  style="cursor: pointer;">Menu</label>
```

Comment:
Avoid inline styles, use CSS classes for it.

Reply:
Fixed in the last commit